tools/antithesis-go-instrumentor/scanning.go

```diff
@@ -0,0 +1,315 @@
+package main
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"go/types"
+	"path"
+	"strconv"
+)
+
+type AntExpect struct {
+	Assertion string
+	Message   string
+	Classname string
+	Funcname  string
+	Receiver  string
+	Filename  string
+	Line      int
+	*AssertionFuncInfo
+}
+
+type AssertionScanningInfo struct {
```

Comment:
Based on how this is used, I'd call it ``AssertionScanner`` instead.

Reply:
Done.